tests/loggers/test_console_logger.py

```diff
@@ -287,3 +288,49 @@ def test_console_logger_with_a_callback(console_logger_test_stream, console_logg
     expected_speed_monitor_lines = num_speed_monitor_lines_per_log_event * expected_num_logging_events
 
     assert actual_num_speed_monitor_lines == expected_speed_monitor_lines
+
+
+@pytest.mark.filterwarnings('ignore:Cannot split tensor of length .* into batches of size .*:UserWarning')
+def test_console_logger_overlapping(console_logger_test_stream, console_logger_test_file_path):
+
+    batch_size = 1
+    dataset_size = 96
```

Comment:
ideally we would make the dataset size smaller so this test is faster. Does the dataset need to be this large for some reason?